src/Storages/System/StorageSystemDatabases.cpp

```diff
@@ -47,7 +49,21 @@ void StorageSystemDatabases::fillData(MutableColumns & res_columns, ContextPtr c
         res_columns[2]->insert(context->getPath() + database->getDataPath());
         res_columns[3]->insert(database->getMetadataPath());
         res_columns[4]->insert(database->getUUID());
-        res_columns[5]->insert(database->getDatabaseComment());
+
+        ASTPtr ast = database->getCreateDatabaseQuery();
+        auto * ast_create = ast->as<ASTCreateQuery>();
+        String engine_full;
+
+        if (ast_create && ast_create->storage) {
+          engine_full = ast_create->storage->formatWithSecretsHidden();
+          static const char *const extra_head = " ENGINE = ";
+
+          if (startsWith(engine_full, extra_head))
```

Comment:
These lines are redundant, formatted ASTStorage does not have `ENGINE = `

Reply:
If delete these code, the result will be like below:
![image](https://user-images.githubusercontent.com/87509571/203686506-b7c12101-7b96-404f-8fb2-396287aebe77.png)
I'm not sure that's what you want.

Reply:
You're right, let's keep this code